### ISSUE TITLE
GA: `setGroupDescription`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2819]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.43.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.42.8...4.43.0) (2022-07-28)

- Release 4.43.0 [`6a98ed3`](https://github.com/open-wa/wa-automate-nodejs/commit/6a98ed3b08e8b15d9ff50c7b904cc51ea4b2c4f1)
- ✨ insiders feat: `sendLocation` now accepts `address` + `url` params #2817 [`0ae073d`](https://github.com/open-wa/wa-automate-nodejs/commit/0ae073dbecbdae5992f071ed7c70c7f7f32a368d)
- 🎁 GA: `autoReject` #2818 [`1ca66f4`](https://github.com/open-wa/wa-automate-nodejs/commit/1ca66f43eb68daf2d0d55da7f0dfb588586da1a9)
- 🩹 fix build issue #2817 [`611dbfc`](https://github.com/open-wa/wa-automate-nodejs/commit/611dbfce968c20a803e0ca67d20a62384bafa5b4)
- 🥚 patches update [`04a7bc3`](https://github.com/open-wa/wa-automate-nodejs/commit/04a7bc3a0b83a403fc71a655b54e614e23a9fdee)
- 🎁 GA: `setGroupDescription` #2819 [`1a4239e`](https://github.com/open-wa/wa-automate-nodejs/commit/1a4239e0ffdf30f25ca0b56cb0ad7b741c8b5b71)
- updated types-only package [`8fa4ffb`](https://github.com/open-wa/wa-automate-nodejs/commit/8fa4ffb57320129a6b9e170d4bf354ece6b681b6)
